pages/common/pngcrush.md

```diff
@@ -0,0 +1,15 @@
+# pngcrush
+
+> PNG image compression utility.
+
+- Compress a PNG file:
+
+`pngcrush {{in.png}} {{out.png}}`
+
+- Compress all PNGs and output to directory:
+
+`pngcrush -d {{path/to/output}} *.png`
+
+- Compress PNG file with best compression:
+
+`pngcrush -rem allb -brute -reduce {{in.png}} {{out.png}}`
```

Comment:
"Best" is a bit vague, and doesn't explain the actual options passed. Care to expand this description a bit?